tests/web/test_outputs.py

```diff
@@ -6,11 +6,16 @@
 OUTPUTS_ENDPOINT = '/api/v1/outputs/'
 
 
-def test_get_outputs_endpoint(client, user_pk):
+@pytest.mark.tendermint
+@pytest.mark.parametrize("bigchain_class", [
+    'bigchaindb.core.Bigchain.get_outputs_filtered',
```

Comment:
We can drop testing `bigchaindb.core.Bigchain.get_outputs_filtered`.

Reply:
We could drop it if we are only testing for tendermint but if we use `mongodb` as backned then we need to test `bigchaindb.core.Bigchain.get_outputs_filtered` as the web view uses this function to prepare the response.